R/colby_constructors.R

```diff
@@ -1827,6 +1828,10 @@ list_wrap_df <- function(f) {
 #' list "100%" as their percentage, even though the individual arm columns
 #' represent strict subsets of the all patients column.
 #'
+#' Note that subtitles [subtitles()] and footers ([main_footer()] and [prov_footer()]) 
```

Comment:
Since there are 2 notes here ("Because percent..." and Note that...") you may want to make them into a bulleted list so the separation of the 2 is clear.